src/main/java/org/soen343/controllers/DashboardController.java

```diff
@@ -26,6 +38,14 @@
     @FXML
     private AnchorPane simulationContext;
 
+    //Info Box Variables
+    @FXML
+    public Text currentDate;
+    public Text currentTime;
+    public Text currentMultiplier;
+    public Slider slider;
```

Comment:
You will also need to add a Slider component to the .fxml file to be able to use that functionality